lib/tbot/tbot_test.go

```diff
@@ -394,3 +404,36 @@ func TestBot_ResumeFromStorage(t *testing.T) {
 	thirdBot := New(botConfig, log)
 	require.NoError(t, thirdBot.Run(ctx))
 }
+
+func TestBot_Insecure(t *testing.T) {
```

Comment:
Nit: We can make it a little clearer what this is testing.

```suggestion
func TestBot_InsecureViaProxy(t *testing.T) {
```